source/github-helpers/api.ts

```diff
@@ -48,6 +48,13 @@ interface RestResponse extends AnyObject {
 	ok: boolean;
 }
 
+export async function log(message: string): Promise<void> {
```

Comment:
We should:

1. move this out of here and into features/index
2. conform its style to the existing logging function — i.e. it should query the options once, not in every call